configure.ac

```diff
@@ -356,6 +356,7 @@ case "$with_platform" in
 	AC_DEFINE([CONFIG_IRQ_MAP], [1], [Configure IRQ maps])
 	AC_DEFINE([CONFIG_DMA_GW], [1], [Configure DMA Gateway])
 	AC_DEFINE([CONFIG_MEM_WND], [1], [Configure Memory Windows])
+	AC_DEFINE([CONFIG_GDB_DEBUG], [1], [Configure GDB debug])
```

Comment:
Sorry, I mean as a stand alone option like
```
# check if we are building ROMs
AC_ARG_ENABLE(roms, [AS_HELP_STRING([--enable-roms],[build roms])], have_roms=$enableval, have_roms=no)
if test "$have_roms" = "yes"; then
	AC_DEFINE([CONFIG_ENABLE_ROMS], [1], [Enable building ROMs for QEMU])
fi
```
That way it can be passed in on the configure cmd line for any platform.

Reply:
@lgirdwood okay, done.